registrations/utils.py

```diff
@@ -104,6 +121,14 @@ def create_event_ics_file_content(event, language="fi"):
     cal.add("prodid", "-//linkedevents.hel.fi//NONSGML API//EN")
     cal.add("version", "2.0")
 
+    # If event name isn't translated for the given language,
+    # try to fall back to a language that has a translation.
+    if (
```

Comment:
I guess modeltranslation would have a builtin way to define fallback languages, but I'm guessing we don't want to add `fallback_languages` to `EventTranslationOptions`?

Reply:
Thinking about this ICS generation function, here it could lead to `event.short_description` being shown in a different language compared to `event.name` (instead of having an empty `event.short_description` in case there is no translation for the same language). It's difficult to say for sure if there are similar risks in other use cases, but it's quite possible due to the global nature of the change.

Reply:
So I tested this for a single event and unlike what I had stated in my previous comment, it seems to work just fine. If there is a field that has value in a fallback language and another field that has a value in another / different fallback language, the first fallback language seems to be used also for the other field (so no chance for having fields in different languages due to fallback differences). So perhaps we just use the built-in `fallback_languages`. Will push that change soon.